packages/reactivity/src/computed.ts

```diff
@@ -16,13 +16,19 @@ export interface WritableComputedRef<T> extends Ref<T> {
 }
 
 export type ComputedGetter<T> = () => T
+export type ComputedGetterWithVModel<T> = (vm: any) => T
```

Comment:
I think this reflects the implementation better than `...args: any[]`. Only 1 argument is passed, and only in options API.

https://github.com/vuejs/core/blob/245230e135152900189f13a4281302de45fdcfaa/packages/runtime-core/src/componentOptions.ts#L692